docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,141 @@
+---
+id: getting-started-linux
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Prerequisites
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
+
+See [Android Setup](/react-native/docs/android-setup.html) for details.
+
+#### Installing NodeJS
+
+The first thing you need to do is to install NodeJS, a popular Javascript implementation.
+
+Fire up the Terminal and paste the following commands to install NodeJS from the [NodeSource](https://nodesource.com/) repository:
+
+```sh
+$ sudo apt-get install -y build-essential
+$ curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+$ sudo ls -s /usr/bin/nodejs /usr/bin/node
+```
+__NOTE__: The above directions are for Ubuntu. If you're on a different distro,  please follow the directions on the [NodeJS website](https://nodejs.org/en/download/).
+
+#### Installing Watchman
+
+[watchman](https://facebook.github.io/watchman/docs/install.html) is a tool by Facebook for watching changes in the filesystem. You need to install it for better performance and avoid a node file-watching bug.
+
+Paste the following into your terminal to compile watchman from source and install it:
+
+```sh
+$ git clone https://github.com/facebook/watchman.git
+$ cd watchman
+$ git checkout v4.1.0  # the latest stable release
+$ ./autogen.sh
+$ ./configure
+$ make
+$ sudo make install
+```
+
+#### Installing Flow
+
+Flow is a static type checker for JavaScript. To install it, paste the following in the terminal:
+
+```sh
+$ wget https://facebook.github.io/flow/downloads/flow-linux64-latest.zip
+$ unzip flow-linux64-latest.zip
+$ cd flow
+$ sudo mv flow /usr/local/bin/flow
+```
+
+__NOTE__: You can put the `flow` executable anywhere on your PATH. It doesn't have to be in `/usr/local/bin`.  (for advanced users)
+
+#### Installing React Native
```

Comment:
This section is already covered by `Quick Start`, which is the next section. Let's remove this, and add info about `sudo` to the `Quick Start`.